operatorcourier/cli.py

```diff
@@ -45,11 +48,18 @@ def parse(self):
         parser.add_argument('-v', '--version',
                             help='Show the current version of operator-courier',
                             action='version', version=__version__)
+        parser.add_argument('--log-level', dest='log_level',
+                            help="Set logging level (default: WARNING)",
+                            choices=['DEBUG', 'INFO', 'WARNING', 'ERROR',
```

Comment:
Do we really want to allow all of these options? Why not just allow DEBUG and INFO? By default we will already log the others.

Reply:
What is wrong about having all of them? when I don't care about WARNINGs I can enable only errors.

I can use `--verbose` or `--debug` to enable DEBUG level, if you insist

Reply:
I would prefer `-v` and `--verbose` to enable DEBUG from the command line since this is a much more common convention.

Reply:
`-v` is already used

I will rework that to `--verbose`